CHANGELOG.rst

```diff
@@ -8,8 +8,8 @@ Changes since 0.4:
 * Documentation corrections.
 
 
-Version 0.4
-===========
+Version 0.4 (2014-10-14)
```

Comment:
Nitpicking: I'd rather put a separate paragraph 'Released 2014-10-14' under the title, rather than putting the date in the title. This would look ugly if the title appears in a table of contents.